.github/workflows/tests.yml

```diff
@@ -1,3 +1,7 @@
+# Copyright © 2016-2020 Endless Mobile, Inc.
+# Copyright © 2020-2024 Endless OS Foundation LLC
```

Comment:
In other projects I have attributed the whole time range to Endless OS Foundation LLC when I update it on the basis that the IP was transferred. Either way seems fine.

Reply:
That does make sense. Lemme update...